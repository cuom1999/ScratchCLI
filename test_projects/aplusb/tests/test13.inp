-2997072969312
7448241668390
