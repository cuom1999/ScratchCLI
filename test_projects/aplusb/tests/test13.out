4451168699078